irisnet/chain.json

```diff
@@ -8,9 +8,7 @@
   "bech32_prefix": "iaa",
   "daemon_name": "iris",
   "node_home": "$HOME/.iris",
-  "key_algos": [
-    "secp256k1"
-  ],
+  "key_algos": ["secp256k1"],
```

Comment:
Can you edit to maintain the original formatting on this and compatible_versions please?

Reply:
Changed back to the original format, thanks @reversesigh

Reply:
why is this duplicating the key_algos property?